hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/GrpcBackendResolver.java

```diff
@@ -35,7 +41,15 @@
           getBackendEntityBuilder(BackendType.GRPC, backendURI.get(), event);
       setAttributeForFirstExistingKey(
           event, entityBuilder, RpcSemanticConventionUtils.getAttributeKeysForGrpcMethod());
-      return Optional.of(entityBuilder.build());
+
+      String grpcMethodAttributeKey =
```

Comment:
missed it last time while reviewing. Can you please add a null check for the attribute key everywhere?